backend/digitales_warten.py

```diff
@@ -153,6 +153,34 @@ def delete_entry(place_id, queue_id, entry_id):
     db.session.commit()
     return ''
 
+@app.route('/places/<place_id>/queues/<queue_id>/entries/<entry_id>', methods=['PUT'])
+def call_entry_from_queue(place_id, queue_id, entry_id):
```

Comment:
you don't use this only to set the CALLED state but to set all states, so rename to `update_...`?

Reply:
Fixed